server/accounts.go

```diff
@@ -2362,44 +2400,33 @@ func (a *Account) checkStreamExportApproved(account *Account, subject string, im
 			if ea == nil {
 				return true
 			}
-			return a.checkAuth(&ea.exportAuth, account, imClaim)
+			return a.checkAuth(&ea.exportAuth, account, imClaim, tokens)
 		}
 	}
 	return false
 }
 
 func (a *Account) checkServiceExportApproved(account *Account, subject string, imClaim *jwt.Import) bool {
 	// Check direct match of subject first
-	se, ok := a.exports.services[subject]
+	ea, ok := a.exports.services[subject]
```

Comment:
Again, would have kept `se` for service export. It just happens that serviceExport embeds an `exportAuth` struct, but really here you get the service export, not the exportAuth.

Reply:
fixed. 
I probably renamed to ea (which is what checkStreamExportApproved named it) to see how similar they are.